### PR TITLE
OT123-353 test endpoints users

### PR DESCRIPTION
**Pull request by facundoveliz**
https://alkemy-labs.atlassian.net/browse/OT123-353

### CODE REVIEW

**Review by DarioMJ48 — APPROVED** [review]

File: controllers/user.js [hunk 1]
```diff
@@ -89,7 +89,7 @@ exports.loginUser = async (req, res) => {
         expiresIn: '1h',
       },
     )
-    res.json({
+    res.status(200).json({
       ok: true,
       msg: 'Login successful',
       result: token,
```
Comment (dvelezroman) [inline controllers/user.js]: `resutl: { user: { ...user }, token }`

**Review by dvelezroman — CHANGES_REQUESTED** [review]
cambiar el response de controller/user.js

**Comment by facundoveliz** [comment]
fixed requested changes @dvelezroman

File: controllers/user.js [hunk 2]
```diff
@@ -89,10 +89,10 @@ exports.loginUser = async (req, res) => {
         expiresIn: '1h',
       },
     )
-    res.json({
+    res.status(200).json({
       ok: true,
       msg: 'Login successful',
-      result: token,
+      result: { user: { ...user }, token },
```
Comment (dvelezroman) [inline controllers/user.js:95]: good

**Review by dvelezroman — APPROVED** [review]

**Pull request merged**